docs/src/test/java/jdbi/doc/StatementsTest.java

```diff
@@ -0,0 +1,66 @@
+package jdbi.doc;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.collection.IsIterableContainingInOrder.contains;
+import static org.junit.Assert.assertThat;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.jdbi.v3.core.H2DatabaseRule;
+import org.jdbi.v3.core.Handle;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class StatementsTest
+{
+    @Rule
+    public H2DatabaseRule db = new H2DatabaseRule();
+
+    private Handle handle;
+
+    @Before
+    public void setUp()
+    {
+        handle = db.getSharedHandle();
+        handle.execute("CREATE TABLE user (id INTEGER PRIMARY KEY, name VARCHAR)");
+        handle.execute("INSERT INTO user VALUES (1, 'Alice')");
+        handle.execute("INSERT INTO user VALUES (2, 'Bob')");
+    }
+
+    @SuppressWarnings("unchecked")
+    @Test
+    public void testQuery() throws Exception
+    {
+        // tag::query[]
+        List<Map<String, Object>> users =
+            handle.createQuery("SELECT id, name FROM user ORDER BY id ASC")
+                .list();
+        assertThat(users, contains(
+                map("id", 1, "name", "Alice"),
+                map("id", 2, "name", "Bob")));
```

Comment:
Can we switch this to assertj?

    assertThat(users).extracting("id").contains(1, 2);
    assertThat(users).extracting("name").contains("Alice", "Bob");